src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -400,8 +405,6 @@ def register_dataset_udf(
             namespace=namespace,
             name=f"{dag_name} registrator ",
             expand_node_output=ingest_list_node,
-            resources=DEFAULT_RESOURCES if resources is None else resources,
```

Comment:
Unless I'm missing something, registration doesn't need special resources. Does this look okay @ktsitsi ?

Reply:
For this one the architecture we are using is dynamic batch taskgraphs. I don't know if changes the resources between the different stages will lead to downscales of running pods or sth. Mainly I had it this way to be sure that the different stages correspond to the same node resources. This is a nice exercise though to test.

Reply:
Indeed though you are right the registration stage is supposed to be lightweight so no special resources are generally needed.